projects/js-packages/config/README.md

```diff
@@ -0,0 +1,29 @@
+# Jetpack Config
+
+When consuming Jetpack JS packages, we need to provide some additional configuration in the webpack config file of our project.
+
+In order to do that, we use the `externals` property:
+
+```JS
+	externals: {
+		...baseConfig.externals,
+		jetpackConfig: JSON.stringify( {
+			plugins_slug: 'my-plugin-slug',
```

Comment:
```suggestion
			plugin_slug: 'my-plugin-slug',
```

Reply:
done, I actually changed it to `consumer_slug` as we discussed